src/main/PDF/PdfMongo.java

```diff
@@ -83,7 +137,10 @@ public static JSONObject getAllFiles(
         res = PdfMessage.SUCCESS.toJSON();
         res.put("documents", files);
       } else if (pdfType == PDFType.FORM) {
-        filter = Filters.eq("metadata.organizationName", organizationName);
+        filter =
+            Filters.and(
+                Filters.eq("metadata.organizationName", organizationName),
+                Filters.eq("metadata.annotated", annotated));
```

Comment:
static import Filters so you can just do

and(eq(), eq()

Reply:
Done